frontend/src/components/MeetingRoom/CaptionsButton/CaptionsBox/SingleCaption/SingleCaption.tsx

```diff
@@ -0,0 +1,80 @@
+import { Subscriber } from '@vonage/client-sdk-video';
+import { ReactElement, useState, useRef, useEffect } from 'react';
+import { Typography } from '@mui/material';
+import useReceivingCaptions from '../../../../../hooks/useReceivingCaptions';
+import { CAPTION_TIMEOUT_MS } from '../../../../../utils/constants';
+
+export type SingleCaptionProps = {
+  subscriber: Subscriber | null;
+  isMobileView: boolean;
+  caption?: string;
+};
+
+/**
+ * SingleCaption component to display captions for a specific subscriber.
+ * @param {SingleCaptionProps} props - The props for the component.
+ * @property {Subscriber} subscriber - The subscriber object for which to display captions.
+ * @returns {ReactElement | null } - The rendered caption or null if not receiving captions.
+ */
+const SingleCaption = ({
+  subscriber,
+  isMobileView,
+  caption,
+}: SingleCaptionProps): ReactElement | null => {
+  const safeSubscriber = subscriber || undefined;
```

Comment:
Do we need this? Could we update the types in `useReceivingCaptions` to account for null instead?